src/network/dandelion.py

```diff
@@ -86,8 +109,8 @@ def hasHash(self, hashId):
         return hashId in self.hashMap
 
     def objectChildStem(self, hashId):
-        """Child (i.e. next) node for an inventory vector during stem mode"""
-        return self.hashMap[hashId].child
+        """Returns Child (i.e. next) node for an inventory vector during stem mode and dandelion enabled status"""
+        return (self.hashMap[hashId].child, self.enabled)
```

Comment:
this isn't necessary, you can retrieve enabled directly as an object attribute

Reply:
https://github.com/Bitmessage/PyBitmessage/pull/2246#discussion_r1618334811 , I've modified it.